{
  "contest": 142,
  "handle": "catchup",
  "address": "0xb159beeb1dea94440820f9a06a2f613974467e0e",
  "risk": "2",
  "title": "batchFillOrder() function is susceptible to DoS",
  "issueId": 83,
  "issueUrl": "https://github.com/code-423n4/2022-06-putty-findings/issues/83"
}